app/helpers/placements/translations_helper.rb

```diff
@@ -1,5 +1,5 @@
 module Placements::TranslationsHelper
   def embedded_link_text(translations)
-    sanitize translations, tags: %w[a]
+    sanitize translations, tags: %w[a], attributes: %w[href target]
```

Comment:
```suggestion
    sanitize translations, tags: %w[a], attributes: %w[href target class]
```

We've chosen to be explicit here, would it be suitable to add the `class` attribute to this list to decrease cognitive load?